cassiopeia/solver/ILPSolver.py

```diff
@@ -272,7 +276,8 @@ def solve(
             cassiopeia_tree.collapse_mutationless_edges(
                 infer_ancestral_characters=True
             )
-        logger.removeHandler(handler)
+
+        #logger.removeHandler(handler)
```

Comment:
why do we have this commented out?

Reply:
Good catch! I had that commented out since I changed `handler` to `file_handler`